README.md

```diff
@@ -66,7 +66,8 @@ The following table lists the software versions NGINX Gateway Fabric supports.
 
 | NGINX Gateway Fabric | Gateway API | Kubernetes | NGINX OSS | NGINX Plus |
 |----------------------|-------------|------------|-----------|------------|
-| Edge                 | 1.2.0       | 1.25+      | 1.27.2    | R32        |
+| Edge                 | 1.2.0       | 1.25+      | 1.27.2    | R33        |
+| 1.5.0                | 1.2.0       | 1.25+      | 1.27.2    | R33        |
 | 1.4.0                | 1.1.0       | 1.25+      | 1.27.1    | R32        |
 | 1.3.0                | 1.1.0       | 1.25+      | 1.27.0    | R32        |
 | 1.2.0                | 1.0.0       | 1.23+      | 1.25.4    | R31        |
```

Comment:
This table is starting to get large. I wonder if we remove entries before 1.0. @kate-osborn @mpstefan thoughts?

Reply:
Agreed, let's remove pre-1.0